kong/clustering/control_plane.lua

```diff
@@ -481,7 +481,10 @@ function _M:handle_cp_websocket()
   self.clients[wb] = queue
 
   if not self.deflated_reconfigure_payload then
-    _, err = self:export_deflated_reconfigure_payload()
+    local ok, err = pcall(self.export_deflated_reconfigure_payload, self)
```

Comment:
this changes the meaning of the `err` variable.  to preserve previous error handling, it should keep both return values in addition of the new `ok`
```suggestion
    local ok, _, err = pcall(self.export_deflated_reconfigure_payload, self)
```

Reply:
https://github.com/Kong/kong/pull/8668/commits/e9142256fc4f7e5f4d4702df6a69a82df9e4cd62